targets/targets.json

```diff
@@ -12525,7 +12525,7 @@
             }
         },
         "components_add": [
-            "FLASHIAP"
```

Comment:
Why the FLASIAP component is removed?

Reply:
Hi,
Before I add the OSPI driver, I have executed the test "features-storage-tests-blockdevice-general_block_device" of FLASHIAP, but it failed.
So, I removed it temporarily to test OSPIF driver.

Reply:
https://github.com/ARMmbed/mbed-os/pull/12619#discussion_r393548847